### PR TITLE
LIRC driver: support for save remote controls via config msg.

### PR DESCRIPTION
**Pull request by hpsaturn**

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -1,6 +1,7 @@
 # Build protocol buffers just to check they compile without errors.
 
 cmake_minimum_required(VERSION 2.8)
+add_definitions(-std=c++11)
```
Comment (arhuaco) [inline CMakeLists.txt]: :-/

It's OK, it's just that there's a better way. This doesn't work with all compilers but we still don't care about those anyway.

File: malos/driver.proto [hunk 2]
```diff
@@ -329,6 +329,9 @@ message LircParams {
 
   // IR command
   string command = 2;
+
+  // Set config
```
Comment (arhuaco) [inline malos/driver.proto]: This is not "set config". Pleas add a more descriptive comment here.

Like:

// IR configuration used for X. It should be in the format XXXX.
// For instance: ZZZZ.

Comment (hpsaturn) [inline malos/driver.proto]: ready, add some documentation similar a nodejs test for this driver.

File: malos/driver.proto [hunk 3]
```diff
@@ -329,6 +329,12 @@ message LircParams {
 
   // IR command
   string command = 2;
+
+  //For transport of LIRC remote devices configuration.
+  //Format: http://www.lirc.org/html/configuration-guide.html
+  //Admobilize config remotes resources:
+  //http://assets.admobilize.com/lirc-remotes/
+  string config = 3;
 }
```
Comment (arhuaco) [inline malos/driver.proto]: Great!

Comment (hpsaturn) [inline malos/driver.proto]: Merge? GPIO documentation PR is pending too

Comment (arhuaco) [inline malos/driver.proto]: Yeah. You can merge this one. Already approved.

**Pull request merged**